### DISCUSSION TITLE
Creating asset subfolders requires right-click which is the only right-click operation that exists in the CP

### DISCUSSION THREAD

**Discussion by ishitatsuyuki** [craftcms/cms#11310] [Ideas] [answered]
Creating asset subfolders can be only done from right-clicking the sidebar entry.

https://craftcms.com/docs/4.x/assets.html#managing-subfolders

Literally nothing else in control panel supports right click, and it makes it virtually impossible to discover without consulting the docs. I think it deserves to be changed to be either a toolbar item or an overflow menu.

**Comment by brandonkelly** [comment, marked as answer]
We’ve addressed this for Craft 4.3 by making those actions available via a new source action menu that’s consistent with the **Fields** and **Sites** index pages (#11906).

**Reply by brandonkelly** [reply]
Craft 4.3 was released earlier this week, with this change!
